editor/editor_resource_picker.cpp

```diff
@@ -184,6 +184,9 @@ void EditorResourcePicker::_update_menu() {
 	Vector2 popup_pos = gt.get_end() - Vector2(ms, 0);
 	edit_menu->set_position(popup_pos);
 	edit_menu->popup();
+	if (has_theme_icon(SNAME("unselect_arrow"), SNAME("EditorResourcePicker"))) {
+		edit_button->set_icon(get_theme_icon(SNAME("unselect_arrow"), SNAME("EditorResourcePicker")));
```

Comment:
```suggestion
	if (has_theme_icon(SNAME("unselect_arrow"))) {
		edit_button->set_icon(get_theme_icon(SNAME("unselect_arrow")));
```
Name of the current class is used implicitly.